packages/Tag/src/Tag.js

```diff
@@ -1,28 +1,38 @@
 import React from "react";
 import PropTypes from "prop-types";
 import RawButton from "@paprika/raw-button";
+import useI18n from "@paprika/l10n/lib/useI18n";
 import * as sc from "./Tag.styles";
 import * as types from "./types";
 
 export default function Tag(props) {
   const { children, onRemove, size, onClick, a11yText, avatar, isDisabled, ...moreProps } = props;
+  const I18n = useI18n();
 
   const handleRemove = e => {
     onRemove();
     e.stopPropagation();
   };
 
+  const handleKeyDown = e => {
```

Comment:
perhaps name this function something that associates it with the Delete Button, maybe `handleDeleteKeyDown`?
Maybe Delete Component should be moved to its own file with export at this point